pkg/manifests/manifests.go

```diff
@@ -74,10 +75,45 @@ const (
 	// ClusterIngressConfigName is the name of the cluster Ingress Config
 	ClusterIngressConfigName = "cluster"
 
-	NamespaceManifest                = "manifests/00-namespace.yaml"
+	// NamespaceManifest is the asset for the operator namespace manifest.
+	// This manifest is usually installed by CVO, but it is also used by the
+	// "ingress-operator render" command to write out manifests that are
+	// required in order to customize ingress at installation time, before
+	// CVO installs the manifests.
+	NamespaceManifest = "manifests/00-namespace.yaml"
+	// CustomResourceDefinitionManifest is the asset for the
+	// ingresscontrollers CRD.  This manifest is usually installed by CVO,
+	// but it is also used by the "ingress-operator render" command to write
+	// out manifests that are required in order to customize ingress at
+	// installation time, before CVO installs the manifests.
 	CustomResourceDefinitionManifest = "manifests/00-custom-resource-definition.yaml"
 )
 
+// Copy needed files from the manifests/ directory.  This is necessary in order
+// to embed these files here because manifests/ is a top-level directory in the
+// source tree, and go:embed does not allow paths containing "../".
+//
+//go:generate mkdir -p manifests
+//go:generate cp ../../manifests/00-namespace.yaml manifests/
+//go:generate cp ../../manifests/00-custom-resource-definition.yaml manifests/
+
+//go:embed assets manifests
+var content embed.FS
+
+// MustAsset returns the bytes for the named asset.
+func MustAsset(asset string) []byte {
```

Comment:
`MustAsset` was called in `MustAssetReader` in the past.  Where is the original definition?

Reply:
Isn't this the https://pkg.go.dev/github.com/abice/go-enum/generator/assets#MustAsset in the `assets` package?

Reply:
`MustAsset` was [defined in `pkg/manifests/bindata.go `](https://github.com/openshift/cluster-ingress-operator/pull/905/files#diff-5ffa278425269cac43b5b2b602dd1ef48f3288d9005e3dec1a652498411c1412L890-L899).